src/bindgen/mangle.rs

```diff
@@ -98,8 +103,8 @@ impl<'a> Mangler<'a> {
         if !self.last {
             self.push(Separator::ClosingAngleBracket)
         }
+        }
```

Comment:
This change looks off, please run rustfmt?

Reply:
Sorted 0f73c70